design-editor/src/pane/element-detector.js

```diff
@@ -11,16 +11,14 @@ function isElementElementSelectable($element, componentPackage, filter) {
 }
```

Comment:
Do you want to commit these changes with actual PR? If yes, please improve this code according to eslint configuration. 
https://travis-ci.org/Samsung/TAU-Design-Editor/jobs/537769906

Reply:
thx, I've fixed the formatting